python/tests/xir/test_integration.py

```diff
@@ -55,6 +55,8 @@ def parse_script(circuit: str) -> XIRProgram:
 g_one(pi) | [0, 1];
 g_two | [2];
 g_three(1, 3.3) | [2];
+adjoint g_four(1.23) | [2];
```

Comment:
I appreciate the consistency here - I would have resorted to just calling these gates `g4` and `g5`.

Reply:
I started with `g_one` and then the rest just followed along.

Reply:
Fair enough!